Description: fix FTBFS due to unused field
 Backport upstream fix.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2016-05-08

---

--- rocksdb-4.5.1.orig/third-party/gtest-1.7.0/fused-src/gtest/gtest.h
+++ rocksdb-4.5.1/third-party/gtest-1.7.0/fused-src/gtest/gtest.h
@@ -17586,7 +17586,7 @@ internal::CartesianProductHolder10<Gener
                       GTEST_TEST_CLASS_NAME_(test_case_name, test_name)>()); \
       return 0; \
     } \
-    static int gtest_registering_dummy_; \
+    static int gtest_registering_dummy_ GTEST_ATTRIBUTE_UNUSED_; \
     GTEST_DISALLOW_COPY_AND_ASSIGN_(\
         GTEST_TEST_CLASS_NAME_(test_case_name, test_name)); \
   }; \
